hack/make/test-docker-py

```diff
@@ -10,8 +10,14 @@ source hack/make/.integration-test-helpers
 : ${DOCKER_PY_COMMIT:=4.0.2}
 
 # custom options to pass py.test
-# TODO remove these skip once we update to a docker-py version that has https://github.com/docker/docker-py/pull/2369 and https://github.com/docker/docker-py/pull/2380
-: ${PY_TEST_OPTIONS:="--deselect=tests/integration/api_swarm_test.py::SwarmTest::test_init_swarm_data_path_addr --deselect=tests/integration/api_exec_test.py::ExecTest::test_detach_with_arg --deselect=tests/integration/api_exec_test.py::ExecDemuxTest::test_exec_command_tty_stream_no_demux"}
+# TODO remove these skip once we update to a docker-py version that has https://github.com/docker/docker-py/pull/2369, https://github.com/docker/docker-py/pull/2380, https://github.com/docker/docker-py/pull/2382
+: ${PY_TEST_OPTIONS:="\
+--deselect=tests/integration/api_swarm_test.py::SwarmTest::test_init_swarm_data_path_addr \
+--deselect=tests/integration/api_exec_test.py::ExecTest::test_detach_with_arg \
+--deselect=tests/integration/api_exec_test.py::ExecDemuxTest::test_exec_command_tty_stream_no_demux \
+--deselect=tests/integration/api_build_test.py::BuildTest::test_build_invalid_platform \
+--deselect=tests/integration/api_image_test.py::PullImageTest::test_build_invalid_platform \
```

Comment:
Made a copy / paste whoopsie here (see https://github.com/moby/moby/pull/39682#issuecomment-520101503, https://github.com/moby/moby/pull/39682/commits/254bb8415c3219fd3b65600e3aff003cd7ef67ac#r312682108, https://github.com/moby/moby/pull/39685#issuecomment-520119396)

Fixing in https://github.com/moby/moby/pull/39714